app/model/SubscriptionData.scala

```diff
@@ -1,15 +1,50 @@
 package model
 
+import com.gu.identity.play.IdUser
+
 case class PaymentData(account: String, sortCode1: String, sortCode2: String, sortCode3: String, holder: String) {
   val sortCode = s"$sortCode1$sortCode2$sortCode3"
 }
+object PaymentData {
+  val blank = PaymentData("", "", "", "", "")
```

Comment:
this value is only used once; can its definition be moved where it is being used?